.env.example

```
@@ -1,6 +1,8 @@
-GITHUB_TOKEN=your_github_token_here
-NEXT_PUBLIC_BASE_URL=http://localhost:3000
-RESEND_AUDIENCE_ID=audience_id
-RESEND_API_KEY=re_xxxx
-NEWSLETTER_SECRET=xxxxx
+# Public site metadata. VITE_ values are exposed to browser code.
+VITE_PUBLIC_BASE_URL=http://localhost:3000
+
+# Newsletter via Resend. Server-side only.
+RESEND_AUDIENCE_ID=
+RESEND_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RESEND_API_KEY key should go before the RESEND_AUDIENCE_ID key

(UnorderedKey)